meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -31,6 +31,9 @@ SDKMACHINE ?= "x86_64"
 # Generate source mirror tarballs:
 BB_GENERATE_MIRROR_TARBALLS = "1"
 
+# Do not allow building of anything that hasn't been checked beforehand.
+SUPPORTED_RECIPES_CHECK = "fatal"
+
 # Configure images in "development mode".
```

Comment:
I disagree. Consider what happens when we make SUPPORTED_RECIPES_CHECK = "fatal" the default: any custom recipe that a user of Ostro adds to his image now needs to be added explicitly to a custom list. That would be a very bad user experience, because most users won't care about controlling their build, at least not initially.

@gvancuts what do you think?